### ISSUE TITLE
Comparison to others does not talk about frozendict 1 vs 2

### ISSUE THREAD

**Issue by gdt** [Harmouch101/frozndict#2]
It's not your fault that frozendict is troubled, but as i understand things:
  - frozendict 2 is in pypi (but doesn't work on 3.11)
  - frozendict 1 sitll exists, not in pypi, last commit in 2016, and seems unmaintained
  
The comparison I think is to frozendict 2.  If so, would be nice to say that.  It would also be nice to talk about frozendict 1 as a similar project.

**Comment by wiseaidev** [comment]
Hey @gdt, thanks for raising this issue. You can take a look at the [similar-projects-comparaison](https://github.com/wiseaidev/frozndict#-similar-projects-comparaison) section for info about the project I was referring to. You can also refer to [this discussion](https://github.com/Marco-Sulla/python-frozendict/issues/57) with the maintainer for more info about immutability. Have a nice day/evening.

**Comment by Marco-Sulla** [comment]
> * frozendict 2 is in pypi (but doesn't work on 3.11)

## TL;DR:
 
`False`. It works for Python 3.6+

## Long explaining:

The ___C extension___ of frozendict (that you call "frozendict 2") works for Python 3.6 - 3.10. The pure Python version works for Python 3.6+. Since pip will install the pure py version if there's no C extension wheel available for your Python and architecture, `frozendict` works for Python 3.6+. You can see in my repo I generate and run more than 260 test cases for all the Python versions, so trust me, it works. If you don't trust me, well, just try it :)

> * frozendict 1 sitll exists, not in pypi, last commit in 2016, and seems unmaintained

Of course the old frozendict, or "frozendict 1", does not exists on pypi. I officially [requested the ownership of the frozendict project](https://github.com/pypi/support/issues/530) since the original author did not respond me privately after 9 months. Only the original github repo of "frozendict 1" still exists, and does not work from Python 3.10.
